conanfile.py

```diff
@@ -1,93 +1,87 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-from conans import ConanFile, CMake, tools
 import os
+from conans import ConanFile, CMake, tools
+from conans.model.version import Version
+from conans.errors import ConanInvalidConfiguration
 
 
 class ProtobufConan(ConanFile):
     name = "protobuf"
     version = "3.6.1"
     url = "https://github.com/bincrafters/conan-protobuf"
     homepage = "https://github.com/protocolbuffers/protobuf"
+    topics = ("protocol-buffers", "protocol-compiler", "serialization", "rpc")
```

Comment:
```suggestion
    topics = ["conan", "protobuf", "protocol-buffers", "protocol-compiler", "serialization", "rpc"]
```

I would recommend that we are always adding `conan` and the libname itself this makes it easier to search through the Conan-verse or for libname specific things. Also you have used `[` in the Conan docs instead of ´(´

Reply:
> I would recommend that we are always adding conan and the libname itself this makes it easier to search through the Conan-verse or for libname specific thing.

Agreed.

> Also you have used [ in the Conan docs instead of ´(´

I also prefer list instead of tuple, but according https://github.com/conan-io/conan/pull/3702#discussion_r223982936 we opted by tuple.

I'll update the topics

Reply:
https://docs.conan.io/en/latest/reference/conanfile/attributes.html?highlight=topics#topics 🤔

Reply:
Ops! outdated documentation. thanks for your eagle eyes!